freeipa-api/src/main/java/com/sequenceiq/freeipa/api/v1/keytab/model/ServiceKeytabRequest.java

```diff
@@ -0,0 +1,70 @@
+package com.sequenceiq.freeipa.api.v1.keytab.model;
+
+import javax.validation.constraints.NotNull;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.sequenceiq.freeipa.api.v1.keytab.doc.KeytabModelDescription;
+import com.sequenceiq.service.api.doc.ModelDescriptions;
+
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+
+@ApiModel("ServiceKeytabV1Request")
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class ServiceKeytabRequest {
+
+    @ApiModelProperty(KeytabModelDescription.ID)
+    private int id;
+
+    @ApiModelProperty(value = ModelDescriptions.ENVIRONMENT_CRN, required = true)
+    private @NotNull String environmentCrn;
```

Comment:
please bring up all `@NotNull` to the same level as `@ApiModelProperty` to ensure the same codestyle

Reply:
It is addressed in https://github.com/hortonworks/cloudbreak/pull/5441